pkg/testutil/testutil.go

```diff
@@ -83,3 +89,119 @@ func (s Setup) Materialize() (workspaceRoot string, err error) {
 
 	return
 }
+
+var dut = flag.Bool("dut", false, "run command/device under test")
```

Comment:
These lines and below were all moved from `pkg/leeway/scripts_test.go` to this file as they're shared between tests.

I don't quite understand what the "dut" is doing, so I'm not sure if there are any unintended consequences of moving this variable to this package.